### DISCUSSION TITLE
Libvips doesn't seems to install on php-fpm alpine image

### DISCUSSION THREAD

**Discussion by stella-mylermedia** [libvips/libvips#2652] [General]
## Bug report

Installing vips for php in an alpine docker image fails after working fine for the past months. The error seems to indicate that vips is not properly installed:
`#18 5.484 configure: error: libvips not found.  Check config.log for more information.
#18 5.506 ERROR: `/tmp/pear/temp/vips/configure --with-php-config=/usr/local/bin/php-config --with-vips' failed`

Seems to be an issue with vips 8.12.2. We managed to temporary resolve it by 8.12.1 with the command below. 
`RUN apk add vips-dev=8.12.1-r0 --update --no-cache --repository http://dl-3.alpinelinux.org/alpine/v3.15/community --repository http://dl-3.alpinelinux.org/alpine/v3.15/main`

**To Reproduce**
1. Use dockerfile

```
FROM php:8.0.15-fpm-alpine

WORKDIR /app

RUN apk update && apk add --no-cache \
    php8-common \
    php8-pdo \
    php8-pdo_mysql \
    php8-mbstring \
    php8-xml \
    php8-openssl \
    php8-json \
    php8-phar \
    php8-zip \
    php8-gd \
    php8-dom \
    php8-session \
    php8-zlib \
    php8-bcmath \
    php8-calendar \
    php8-exif \
    php8-ffi \
    php8-gettext \
    php8-gmp \
    php8-imap \
    php8-soap \
    php8-sockets \
    php8-sysvmsg \
    php8-sysvshm \
    php8-xsl \
    php8-intl \
    php8-zip \
    php8-shmop \
    autoconf \
    gcc \
    libc-dev \
    pkgconf \
    make \
    libmcrypt \
    libmemcached \
    php8-pecl-memcached \
    npm \
    libzip-dev \
    zip \
    imagemagick \
    imagemagick-dev \
    libtool

RUN apk add vips-dev --update --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community --repository http://dl-3.alpinelinux.org/alpine/edge/main

RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer

RUN docker-php-ext-install pdo pdo_mysql
RUN docker-php-ext-enable pdo_mysql
RUN docker-php-ext-install pcntl

RUN pecl config-set php_ini  /usr/local/etc/php/php.ini

RUN printf "\n" | pecl install imagick msgpack igbinary redis zip
RUN printf "\n" | pecl install vips
```

2. Build the dockerfile
3. See error

**Expected behavior**
I expected that pecl install vips would install vips.

**Actual behavior**
pecl install vips fails. 

**Environment**
OS and Docker version that are used. 
 - OS: MacOS 11.6.2
 - Docker version 20.10.8, build 3967b7d

**Comment by lovell** [comment]
> RUN apk add vips-dev --update --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community --repository http://dl-3.alpinelinux.org/alpine/edge/main

It looks like you're trying to install "edge" packages on a non-edge version of Alpine.

https://wiki.alpinelinux.org/wiki/Edge
